src/pages/PrintOnlyScreen.tsx

```diff
@@ -62,7 +72,15 @@ const PrintOnlyScreen = ({
     !isPrinted
 
   const printBallot = useCallback(async () => {
-    const ballotTrackingCode = await encryptBallot(votes)
+    const ballotTrackingCode = await encryptBallot({
+      election,
+      ballotStyle: getBallotStyle({ ballotStyleId, election }),
+      precinct: getPrecinctById({ precinctId, election })!,
+      ballotId,
+      votes,
+      isTestBallot: !isLiveMode,
+      ballotType: 0,
```

Comment:
This is an enum value, right? Can we use `BallotType.Normal` or whatever this is?